tox.ini

```diff
@@ -1,13 +1,11 @@
-[tox]
-envlist = py27, py27-scrapy1.0.x, py35, py36
-
-[testenv]
-deps =
-    -r{toxinidir}/requirements.txt
-    -r{toxinidir}/requirements-dev.txt
-commands = py.test {posargs}
-
-[testenv:py27-scrapy1.0.x]
-deps =
-    Scrapy<1.1
-    -r{toxinidir}/requirements-dev.txt
+[tox]
+envlist = py27, py27-scrapy{1.0,1.1,1.2,1.3}, py35, py36, {py35,py36}-scrapy{1.1,1.2,1.3}
```

Comment:
It's for up-to-date versions (when there's Scrapy>1.3), but today it's indeed equivalent to using scrapy1.3